### PR TITLE
Stripe / React

### PR DESCRIPTION
**Pull request by reactivepixel**

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -20,9 +20,13 @@
...
         <h2 class="bg-success">Bootstrap connected</h2>
         <a role="button" href="/status" class="btn btn-primary btn-lg">VIEW STATUS ROUTE</a>
+
+        <div id="app"></div>
+        <script src="bundle.js"></script>
+
       </div>
+    </div>
   </div>
-</div>
```
Comment (reactivepixel) [inline dist/index.html]: Make certain this is a buildableish through gulp.

File: package.json [hunk 2]
```diff
@@ -15,20 +15,40 @@
...
     "gulp": "^3.9.1",
     "gulp-nodemon": "^2.0.6",
     "gulp-shell": "^0.5.2",
+    "jsdom": "^8.4.0",
     "mocha": "^2.4.5",
-    "supertest": "^1.2.0"
+    "react-hot-loader": "^1.3.0",
+    "style-loader": "^0.13.1",
+    "supertest": "^1.2.0",
+    "webpack": "^1.13.0",
```
Comment (reactivepixel) [inline package.json]: No Carrots, we're not horses

File: src/components/StoreApp.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+      </div>
+      </li>
+    )}
+    </ul>
+    </section>
+    </section>
+    </div>
+  }
+};
```
Comment (reactivepixel) [inline src/components/StoreApp.jsx]: /src/client/**

File: src/routes/index.js [hunk 4]
```diff
@@ -12,12 +12,12 @@ module.exports = function(express) {
...
-      msg: 'Hello World',
-      healthy: true
-    });
-  });
+  // router.get('/', function(req, res) {
+  //   res.status(200).json({
+  //     msg: 'Hello World',
+  //     healthy: true
+  //   });
+  // });
```
Comment (reactivepixel) [inline src/routes/index.js]: Remove unused code

File: src/routes/index.js [hunk 5]
```diff
@@ -26,10 +26,23 @@ module.exports = function(express) {
...
+  router.get('/react', function(req,res){
+    res.sendFile(__dirname + './../../dist/index.html');
+  });
+
+  router.get('/', function(req,res){
+    res.sendFile(__dirname + './../../dist/index.html');
+  });
 
+  router.get('/charge', function(req,res){
+    res.sendFile(__dirname + '/public/charge.html');
```
Comment (reactivepixel) [inline src/routes/index.js]: Swap to express.static

File: src/routes/public/charge.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+      data-amount="999"
+      data-name="Demo Site"
+      data-description="Widget"
+      data-image="/img/documentation/checkout/marketplace.png"
+      data-locale="auto">
+      </script>
+    </form>
+  </div>
+</body>
+</html>
```
Comment (reactivepixel) [inline src/routes/public/charge.html]: Add public and other build folders to gitignore.

File: package.json [hunk 7]
```diff
@@ -15,35 +15,35 @@
...
   },
   "devDependencies": {
-    "babel-core": "^6.7.7",
-    "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "chai": "^3.5.0",
-    "chai-immutable": "^1.5.4",
-    "css-loader": "^0.23.1",
-    "dotenv": "^2.0.0",
```
Comment (jc-roll) [inline package.json:26]: Scary I get why but are we sure? No carrots #53

**Pull request closed without merging**